### ISSUE TITLE
Checkout.PaymentCaptureRequest doesn't have the AdditionalData dictionary

### ISSUE THREAD

**Issue by milandhango** [Adyen/adyen-dotnet-api-library#990]
https://github.com/Adyen/adyen-dotnet-api-library/blob/main/Adyen/Model/Checkout/PaymentCaptureRequest.cs does not have the AdditionalData dictionary that allows us to pass in the Level 2/3 data on a capture per the documentation (https://docs.adyen.com/payment-methods/cards/enhanced-scheme-data/l2-l3/).

The dictionary is present on the Adyen.Model.Checkout.PaymentRequest model.

Can you please add that? Or let me know if there is another way to submit that data on a capture?

**Comment by DjoykeAbyah** [comment]
Hi @milandhango! 

My apologies for the delayed reply! Maybe you've already found a workaround or solution yourself but if you use [this]https://github.com/Adyen/adyen-dotnet-api-library/blob/main/Adyen/Model/Payment/CaptureRequest.cs#L74) class it would solve your issue. 

Regards,

Djoyke
Adyen

**Comment by milandhango** [comment]
My understanding is that we are supposed to be using the models from the Checkout namespace for consistency. For captures, we are using Adyen.Model.Checkout.PaymentCaptureRequest. However, the link you sent is to Adyen.Model.Payment.CaptureRequest. Can you add the AdditionalData property to Adyen.Model.Checkout.PaymentCaptureRequest?

**Comment by DjoykeAbyah** [comment]
You're right, @milandhango. I suggested the wrong model. My apologies for the mistake. I'll forward this question to the API owners and get back to you once I have more information.

**Comment by DjoykeAbyah** [comment]
Hi @milandhango,

I've contacted the API owners regarding your issue. You can use the temporary workaround I mentioned earlier until the AdditionalData field is added to PaymentCaptureRequest. Once this feature is implemented, I will give you a heads-up!

Thank you for your patience.

Best regards,
Djoyke
Adyen

**Comment by milandhango** [comment]
I'm being told this is not right since the totalTaxAmount should be in the enhancedSchemeData model and not nested under the Level23. Philip Galang is the one who mentioned this.

**Comment by milandhango** [comment]
To add more context - we are seeing an error in sandbox (see below).


{
  "live": "false",
  "notificationItems": [
    {
      "NotificationRequestItem": {
        "additionalData": {
          "metadata.accountId": "123",
          "enhancedSchemeDataReceived": "L3",
          "metadata.userId": "jsmith",
          "enhancedSchemeDataRefusalReasons": "enhancedSchemeData.totalTaxAmount is required",
          "bookingDate": "2026-05-29T03:44:18Z",
          "enhancedSchemeDataWarningReasons": "Destination country code is required when destination postal code is present, Destination postal code is invalid, Ship from postal code is invalid",
          "enhancedSchemeDataSubmitted": ""
        },
        "amount": {
          "currency": "USD",
          "value": 15000
        },
        "eventCode": "CAPTURE",
        "eventDate": "2026-05-29T05:43:46+02:00",
        "merchantAccountCode": "SummitPay_Platform_US_TEST",
        "merchantReference": "57ae7d4a5a804043ac0d8a5c8359332d",
        "originalReference": "XD64GCSD98KPT9V5",
        "paymentMethod": "visa",
        "pspReference": "SC9FZ94BHB3VZTV5",
        "reason": "",
        "success": "true"
      }
    }
  ]
}
